test/javascripts/acceptance/close-chat-channel-modal-test.js

```diff
@@ -0,0 +1,42 @@
+import { acceptance } from "discourse/tests/helpers/qunit-helpers";
```

Comment:
This should be `delete-chat-channel-modal-test` I think, also will we add the other tests for it back in at some point from that old delete test that was removed recently?